src/process/intermediate.rs

```diff
@@ -60,6 +63,17 @@ pub fn container_intermediate(
             .with_context(|| format!("Failed to enter pid namespace: {:?}", pid_namespace))?;
     }
 
+    // this needs to be done before we create the init process, so that the init
+    // process will already be captured by the cgroup
+    if args.rootless.is_none() {
+        apply_cgroups(
```

Comment:
Can I ask you to update  [docs/.drawio.svg](https://github.com/containers/youki/blob/main/docs/.drawio.svg)  using  [vscode-drawing](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).